azure-pipelines.yml

```diff
@@ -12,7 +12,7 @@ variables:
   DotNetVersion: 6.0.300
   DotNet6Source: https://aka.ms/dotnet6/nuget/index.json
   NuGetOrgSource: https://api.nuget.org/v3/index.json
-  XamarinDotNetWorkloadSource: https://aka.ms/dotnet/maui/rc.3.json
+  XamarinDotNetWorkloadSource: https://aka.ms/dotnet/maui/6.0.400.json
```

Comment:
This needs to use the local file below? But can you update .NET & workloads in a separate PR? Just to make sure that doesn't break anything on its own.